_includes/graphql/customer-payment-tokens.md

```diff
@@ -0,0 +1,16 @@
+The `CustomerPaymentTokens` output object contains an array of `items`.
+
+Attribute | Data Type | Description
+--- | --- | ---
+`items` | [PaymentToken] | Contains an array of customer payment tokens
+
+#### PaymentToken attributes
+
+The `PaymentToken` object defines characteristics of a token stored in the payment vault.
+
+Attribute | Data Type | Description
+--- | --- | ---
+`details` | String | Stored account details
+`payment_method_code` | String | The payment method code associated with the token
+`public_hash` | String | The public hash of the token
```

Comment:
Should we mention this is automatically generated like an auth token or customer cart ID? Maybe that's only necessary if we add a mutation to add payment token.

Reply:
Mentioned that the vault provider generates the token